lib/components/user/user-account-screen.tsx

```diff
@@ -193,45 +194,30 @@ class UserAccountScreen extends Component<Props> {
               // We pass the Formik props below to the components rendered so that individual controls
               // can be wired to be managed by Formik.
               (formikProps) => {
-                if (itemId === 'verify') {
-                  const verifyEmail = intl.formatMessage({
-                    id: 'components.NewAccountWizard.verify'
-                  })
-                  return (
-                    <Form id="user-settings-form" noValidate>
-                      <PageTitle title={verifyEmail} />
-                      <h1>{verifyEmail}</h1>
-                      <VerifyEmailPane />
-                    </Form>
-                  )
-                }
-
                 const newFormikProps = {
                   ...formikProps,
                   // Use our own handleChange handler that wraps around Formik's.
                   handleChange: this._handleInputChange(formikProps)
                 }
-
-                if (basePath === CREATE_ACCOUNT_PATH) {
-                  return (
-                    <NewAccountWizard
-                      activePaneId={itemId}
-                      formikProps={newFormikProps}
-                      onCreate={this._handleCreateNewUser}
-                    />
-                  )
-                }
-
-                if (basePath === MOBILITY_PATH) {
-                  return (
-                    <MobilityWizard
-                      activePaneId={itemId}
-                      formikProps={newFormikProps}
-                    />
-                  )
-                }
-
-                return <ExistingAccountDisplay {...newFormikProps} />
+                return (
+                  <Switch>
+                    <Route exact path={CREATE_ACCOUNT_VERIFY_PATH}>
+                      <VerifyEmailScreen />
+                    </Route>
+                    <Route path={CREATE_ACCOUNT_PATH}>
+                      <NewAccountWizard
+                        formikProps={newFormikProps}
+                        onCreate={this._handleCreateNewUser}
+                      />
+                    </Route>
+                    <Route path={MOBILITY_PATH}>
+                      <MobilityWizard formikProps={newFormikProps} />
+                    </Route>
+                    <Route>
+                      <ExistingAccountDisplay {...newFormikProps} />
+                    </Route>
+                  </Switch>
```

Comment:
I think this change is causing the entire page to refresh when clicking some links in the account screen, but it's a worthy trade off in exchange for the benefits this PR provides!